### DISCUSSION TITLE
Accessing data from response

### DISCUSSION THREAD

**Discussion by DevendarKommidi** [postmanlabs/postman-flows#366] [Defect]
Hello,

I have created a simple flow, which will fetch the response of type list. The data looks like this after using select block.
<img width="1060" alt="fig1" src="https://user-images.githubusercontent.com/89071750/183524809-b4ca99ca-160c-4d6d-9638-a29caf3f935a.png">

Now I want to access the data which is inside the list. I have tried in different ways but not able to loop the data. Could you please help with this?

One of the way tried is by calling the another select block. But I am getting data as null. As shown below.
<img width="1029" alt="fig2" src="https://user-images.githubusercontent.com/89071750/183525176-14effca2-f353-40db-bafc-f8d5e6127524.png">

**Comment by ghost** [comment]
Hello,

Did you save an example of your request? See: [Create an example](https://github.com/postmanlabs/postman-flows/discussions/358)

Once you have created an example, you should be able to drill into the response body to retrieve individual variables via a block's "variables" interface.

Side-note: It would be really awesome if someone could update the docs here: https://learning.postman.com/labs/postman-flows/getting-started/chaining-requests-with-data/

This is an incredibly opaque requirement for making variable access work...

**Reply by DevendarKommidi** [reply]
Thanks for your replay. I have saved as an example and tried. But the body is still showing as string and I am not able to access the data inside the body. Attaching the screenshots for your reference.

<img width="1021" alt="Screenshot 2022-08-13 at 21 44 03" src="https://user-images.githubusercontent.com/89071750/184510235-5b99f840-5583-4186-bb0b-56200169ace2.png">

And my saved response looks as below.

<img width="1015" alt="Screenshot 2022-08-13 at 21 48 14" src="https://user-images.githubusercontent.com/89071750/184510294-7f354409-52e0-4b99-9c82-99d2add1cb81.png">


I need to access hash from the list. 

Thanks in Advance.
